content/work/shopify.md

```diff
@@ -29,5 +33,47 @@ summary: |
   with a follow-up Q&A.
 ---
 
+{% import 'embed.macros.njk' as embed %}
+{% import 'layout.macros.njk' as layout %}
 {% import 'quotes.macros.njk' as quotes %}
-{{ quotes.blockquote(press[0]) }}
+
+{{ layout.title('What We Did') }}
+
+{% call embed.media_block(
+  media=embed.screen(
+    notch='0',
+    content=embed.img(
+      src='pages/shopify-phone.jpg',
```

Comment:
Is this a helpful image? It doesn't make sense to me -- why is Miriam training them on a mobile phone? 😄

Reply:
Sondra swapped images here:
![bild](https://user-images.githubusercontent.com/41588129/132854541-4340fa69-0942-457e-9182-326ac38e8299.png)